packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -1785,4 +1785,75 @@ describe("TableComponent", () => {
+			{ id: 3, description: "dummy 3", test_one: "test-1", test_two: "test-2", info: "info-3", more_info: "more-info-3" }
+		];
+
+		beforeEach(() => {
+			hostComponent.columnProperties = [
+				{ name: "id", sticky: false },
+				{ name: "description", sticky: true },
+				{ name: "test_one" },
+				{ name: "test_two"},
+				{ name: "info", stickyEnd: false },
+				{ name: "more_info", stickyEnd: true }
+			];
+			hostComponent.dummyData = dummyData;
+
+			hostFixture.detectChanges(); // trigger data binding
+			component.ngAfterViewInit();
+		});
+		
+		it("should have class mat-table-sticky when sticky is true", () => {
+			const cellElements: HTMLElement[] = hostFixture.nativeElement.querySelectorAll("table tbody tr td.mat-column-description");
+			expect(cellElements.length).toBe(3);
+			cellElements.forEach((cellElement: HTMLElement) => {
+				expect(cellElement && cellElement.classList).toContain("mat-table-sticky");
+			});
+		});
+
+		it("should have class mat-table-sticky when stickyEnd is true", () => {
+			const cellElements: NodeListOf<HTMLElement> = hostFixture.nativeElement.querySelectorAll("table tbody tr td.mat-column-more_info");
+			expect(cellElements.length).toBe(3);
+			cellElements.forEach((cellElement: HTMLElement) => {
+				expect(cellElement && cellElement.classList).toContain("mat-table-sticky");
+			});
+		});
+
+		it("should not have class mat-table-sticky when sticky is false", () => {
+			const cellElements: NodeListOf<HTMLElement> = hostFixture.nativeElement.querySelectorAll("table tbody tr td.mat-column-id");
+			expect(cellElements.length).toBe(3);
+			cellElements.forEach((cellElement: HTMLElement) => {
+				expect(cellElement && cellElement.classList).not.toContain("mat-table-sticky");
+			});
+		});
+
+		it("should not have class mat-table-sticky when stickyEnd is false", () => {
+			const cellElements: NodeListOf<HTMLElement> = hostFixture.nativeElement.querySelectorAll("table tbody tr td.mat-column-info");
+			expect(cellElements.length).toBe(3);
+			cellElements.forEach((cellElement: HTMLElement) => {
+				expect(cellElement && cellElement.classList).not.toContain("mat-table-sticky");
+			});
+		});
+
+		it("should not have class mat-table-sticky when sticky is not defined", () => {
+			const cellElements: NodeListOf<HTMLElement> = hostFixture.nativeElement.querySelectorAll("table tbody tr td.mat-column-test_one");
+			expect(cellElements.length).toBe(3);
+			cellElements.forEach((cellElement: HTMLElement) => {
+				expect(cellElement && cellElement.classList).not.toContain("mat-table-sticky");
+			});
+		});
+
+		it("should not have class mat-table-sticky when stickyEnd is not defined", () => {
+			const cellElements: NodeListOf<HTMLElement> = hostFixture.nativeElement.querySelectorAll("table tbody tr td.mat-column-test_two");
+			expect(cellElements.length).toBe(3);
+			cellElements.forEach((cellElement: HTMLElement) => {
+				expect(cellElement && cellElement.classList).not.toContain("mat-table-sticky");
```

Comment:
Could you please adapt your test, to avoid this duplicate string warning ?
Thanks